Makefile

```diff
@@ -5,31 +5,27 @@ SHELL=/bin/bash -o pipefail
 #          $(if $(shell which $(exec)),some string,$(error "No $(exec) in PATH")))
 
 export GO111MODULE        := on
-export PATH               := .bin:${PATH}
-export PWD                := $(shell pwd)
+export CWD                := $(shell pwd)
 export BUILD_DATE         := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 export VCS_REF            := $(shell git rev-parse HEAD)
 export QUICKSTART_OPTIONS ?= ""
 
-GO_DEPENDENCIES = github.com/ory/go-acc \
-				  github.com/golang/mock/mockgen \
-				  github.com/go-swagger/go-swagger/cmd/swagger \
-				  golang.org/x/tools/cmd/goimports \
-				  github.com/mattn/goveralls \
-				  github.com/cortesi/modd/cmd/modd
```

Comment:
The CI now fails with: https://github.com/ory/kratos/actions/runs/3423785129/jobs/5702805028#step:16:53

Can you please re-add the tooling included here? :)

Reply:
Sorry, this must be a sync error. These changes were not supposed to be in here.